spec/game_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'game' 
+require 'board'
+
+describe Game do
+  EMPTY_BOARD = Board.empty_board
+
+  context 'new game' do
+    it 'has an empty board' do
```

Comment:
Is it alright for game to exist and have no tests if it is used in another class like ConsoleDisplay?